src/App.tsx

```diff
@@ -1,18 +1,45 @@
 import React from 'react';
+import { Tabs } from './Tabs';
 
 import './App.scss';
 
 const tabs: Tab[] = [
-  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+  { id: 'tab-1', title: 'Tab 1', content: 'TEXT 1 - Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque maxime excepturi quaerat atque asperiores hic? Aliquam adipisci autem labore nulla nemo distinctio vel beatae minima, magni quam suscipit nisi neque.' },
+  { id: 'tab-2', title: 'Tab 2', content: 'TEXT 2 - Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque maxime excepturi quaerat atque asperiores hic? Aliquam adipisci autem labore nulla nemo distinctio vel beatae minima, magni quam suscipit nisi neque.' },
+  { id: 'tab-3', title: 'Tab 3', content: 'TEXT 3 - Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque maxime excepturi quaerat atque asperiores hic? Aliquam adipisci autem labore nulla nemo distinctio vel beatae minima, magni quam suscipit nisi neque.' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+interface State {
+  selectedTab: Tab;
+}
+
+export class App extends React.Component<{}, State> {
+  state: State = {
+    selectedTab: tabs[0],
+  };
+
+  chooseTab = (tab: Tab) => (
+    this.setState({
+      selectedTab: tab,
+    })
+  );
+
+  render() {
+    return (
+      <div className="App">
+        <h1>
+          {'Selected tab is '}
+          {this.state.selectedTab.title}
```

Comment:
```suggestion
          {`Selected tab is ${this.state.selectedTab.title}`}
```
and better make destructuring